File: praktikum_algopascal/pascal/luasSegitiga.pas
// luas segitiga L = 2 * (1/2 * a * t)

program luasSegitiga;
	uses crt;
		function LS(x, y: Integer): real;

begin
	LS:= 1/2 * (x*y);
end;

	var
		a, t: Integer;
begin
	writeln('Program Luas Segitiga');
	write('Masukan Nilai alas: ');
	readln(a);
	write('Masukan Nilai tinggi: ');
	readln(t);

	writeln('=======================');
	write('Hasil Luas Segitiga: ', LS(a, t):1:0);
	readln;
end.

File: praktikum_algopascal/C/LuasSegitiga.c
	// luas segitiga = L = 2 * (1/2 * a * t)

#include <stdio.h>
int luasSegitiga(int a, int t)
{
	int luasnya = 1/2 * a * t;
	//int luasnya = 2 * (1/2 * a * t);
}

int main()
{
	int a, t;
	printf("Masukan Nilai alas : ");
	scanf("%d", &a);
	printf("Masukan Nilai Tinggi: ");
	scanf("%d", &t);
	printf("===================\n");
	printf("Hasil Luas Segitiga : %d", luasSegitiga(a, t));

	return 0;
}